### PR TITLE
Radiomaster er4 er6

### PR DESCRIPTION
**Pull request by brandonrc**
Adding following targets:

- Radiomaster ER4 RX
- Radiomaster ER6-GV RX

### CODE REVIEW

File: src/hardware/targets.json [hunk 1]
```diff
@@ -1306,17 +1306,35 @@
...
                 "upload_methods": ["uart", "wifi", "etx"],
                 "platform": "esp32",
                 "firmware": "Unified_ESP32_2400_TX"
             }
         },
         "rx_2400": {
+            "er4": {
+                "product_name": "RadioMaster ER4 2.4GHz PWM RX",
+                "lua_name": "RM ER4",
+                "layout_file": "Generic 2400 PWMP5.json",
```
Comment (pkendall64) [inline src/hardware/targets.json:1320]: Should also have
```
                "overlay": {
                    "power_dcdc": true,
                    "pwm_outputs": [0,1,3,9]
                },
```

File: src/hardware/targets.json [hunk 2]
```diff
@@ -1306,17 +1306,35 @@
...
+                "upload_methods": ["uart", "wifi"],
+                "platform": "esp8285",
+                "firmware": "Unified_ESP8285_2400_RX",
+                "prior_target_name": "DIY_2400_RX_PWMP"
+            },
+            "er6": {
+                "product_name": "RadioMaster ER6-GV 2.4GHz PWM RX",
+                "lua_name": "RM ER6-GV",
                 "layout_file": "Generic 2400 PWMP5.json",
```
Comment (pkendall64) [inline src/hardware/targets.json:1338]: Should also have
```
                "overlay": {
                    "power_dcdc": true
                },
```

File: src/hardware/targets.json (hunk 1 above)
Comment (brandonrc) [reply src/hardware/targets.json:1320]: Fixed

File: src/hardware/targets.json (hunk 2 above)
Comment (brandonrc) [reply src/hardware/targets.json:1342]: Fixed!

**Comment by brandonrc** [comment]
@pkendall64 was this something we wanted to add to 3.2? Or push to the next release?

**Review by pkendall64 — APPROVED** [review]
I'm ok if they make into 3.2, not sure they've been release yet.

**Review by deadbytefpv — APPROVED** [review]

**Pull request merged**